File: AppLovin-Android-Demo-App/src/main/java/com/applovin/apps/demoapp/leaders/LeaderDemoMenuActivity.java
package com.applovin.apps.demoapp.leaders;

import android.content.Intent;

import com.applovin.apps.demoapp.DemoMenuActivity;
import com.applovin.apps.demoapp.DemoMenuItem;

public class LeaderDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Programmatic", "Programmatically create an instance of it", new Intent( this, LeaderProgrammaticActivity.class ) ),
                new DemoMenuItem( "Layout Editor", "Create a Leader from the layout editor", new Intent( this, LeaderLayoutEditorActivity.class ) ),
        };
        return result;
    }
}

File: Android-Kotlin-Demo-App/src/main/java/com/applovin/apps/kotlindemoapp/leaders/LeaderDemoMenuActivity.kt
package com.applovin.apps.kotlindemoapp.leaders

import android.content.Intent
import com.applovin.apps.kotlindemoapp.DemoMenuActivity
import com.applovin.apps.kotlindemoapp.DemoMenuItem

class LeaderDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Programmatic", "Programmatically create an instance of it", Intent(this, LeaderProgrammaticActivity::class.java)),
            DemoMenuItem("Layout Editor", "Create a Leader from the layout editor", Intent(this, LeaderLayoutEditorActivity::class.java))
    )
}
